docs/docs/guides/authentication/cloudentity.md

```diff
@@ -0,0 +1,110 @@
+# Cloudentity
+
+Kusk Gateway enables using third-party authentication and authorization management solutions such as Cloudentity.
+
+Developers can utilize Cloudentity's platform to control their authorization flows while still using OpenAPI with Kusk to configure gateway level rules.
+
+This separation of concerns is important when your authorization flow complexity grows or the number of services talking to each other require more authorization granularity.
+
+## Setup guide
+
+By the end of this guide, you will have deployed an API to Kusk and used Cloudentity to block requests to your API. 
+
+### 1. Create Kusk Authorizer in Cloudentity
+
+On Cloudentity's platform, go to **Enforcement > Authorizers**, click **CREATE GATEWAY**, and from the list that's displayed choose Kusk.
+
+![cloudentity authorizers list](./img/cloudentity-authorizers.png)
+
+### 2. Install Cloudentity's authorizer in your cluster
+
+The bottom of the quickstart section contains a `helm` command that installs Cloudentity's authorizer and configures with credentials that are used to pull authorization policies from Cloudentity. 
```

Comment:
It reads better as:

The bottom of the quickstart section contains a `helm` command that installs Cloudentity's authorizer and configures _it_

Missing _it.